versions/v6/static/scss/pages/_archives.scss

```diff
@@ -0,0 +1,147 @@
+// Archives
+#archives {
```

Comment:
We should ideally update this at some point to not reference an ID selector here (`#archives`)